tests/memory_checker/test_memory_checker.py

```diff
@@ -692,8 +737,7 @@ def test_memory_checker_without_container_created(duthosts, enum_dut_feature_con
     container_name = "telemetry"
 
     pytest_require("Celestica-E1031" not in duthost.facts["hwsku"]
-                   and (("20191130" in duthost.os_version and parse_version(duthost.os_version) > parse_version("20191130.72"))
-                   or parse_version(duthost.kernel_version) > parse_version("4.9.0")),
-                   "Test is not supported for platform Celestica E1031, 20191130.72 and older image versions!")
+                   and ("20201231" in duthost.os_version or parse_version(duthost.kernel_version) > parse_version("4.9.0")),
```

Comment:
@yozhao101 We are in the path to move all skip to tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
which will bring us some benefits. 
1. Reduce the test time by skip the test at the beginning.
2. Reduce unnecessary test result, some test cases should be skipped but report error during setup stage.

could you help to move this skip to conditional mark file?

Reply:
Sure.

Reply:
@StormLiangMS @wangxin I updated the PR and can you please help me review?